spec/support/authentication_helpers.rb

```diff
@@ -1,25 +1,27 @@
 # frozen_string_literal: true
 
 module AuthenticationHelpers
-  mattr_accessor :username, :password
+  mattr_accessor :username, :password, :token
```

Comment:
We already used mattrs in this module... why not use one more if it lets us ditch our only Rubocop TODO?